### ISSUE TITLE
when i install this when i join a lobby i cant leave it for some reason

### ISSUE THREAD

**Issue by devon749** [developer9998/HoldablePad#7]

**Comment by developer9998** [comment]
This issue will be reviewed later this week.

**Comment by devon749** [comment]
its more like i cant join one after i left one ,sorry for the mistake

**Comment by devon749** [comment]
would you like me to send a vid

**Comment by developer9998** [comment]
Nope, thanks though.

**Comment by devon749** [comment]
ok

**Comment by dualposition** [comment]
this might be an issue with other mods
i had this issue with computer interface

**Comment by dualposition** [comment]
this also might be a glitch with tmploader
some people had a glitch with tmploader
that they cant join

**Comment by developer9998** [comment]
Thats correct - TMPLoader is now obsolete. I wouldn't know how of any issues caused by it though.

**Comment by devon749** [comment]
im gonna delete tmp loader thx

**Comment by devon749** [comment]
im gonna try holdable pad rn without it

**Comment by devon749** [comment]
nvm it didnt fix

**Comment by dualposition** [comment]
nvm its a problem with holdablepad
i dont have tmploader or CI

**Comment by devon749** [comment]
the shirts mod works so i know its not a problem caused by something he intended to do

**Comment by devon749** [comment]
is there any updates on a fix?

**Comment by developer9998** [comment]
As I've already stated, this issue will be reviewed later this week. Please have some patience; I have prioritized other work besides this project.
